tests/test_filters.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+from ac_dc.filters import *
+
+
+strip_characters = (
+    "' 0123456789¯_%$§½¼¾×|†~\"—±′–'°−{}[]·'?,./<>!@#^&*()+-‑=:;，：,...�`→¶.…‘’”"
+)
+
+special_characters = (
+    """\' 0123456789¯_%$§½¼¾×|†~"—±′–\'°−{}[]·\'?,./<>!@#^&*()+-‑=:;，：,...�`→¶.…‘’”'"""
+)
+
+
+def test_lower_strip_sentence():
+    assert lower_strip_sentence(" Hello World   ") == "hello world"
```

Comment:
Great to have these sanity checks.
We need ASAP to add test cases for all languages and ensure we have correct parsing for all.